### ISSUE TITLE
Add a Sphinx plot gallery

### ISSUE THREAD

**Issue by cdeil** [astroplanners/astroplan#12]
@jberlanga, @adrn, all – What do you think about adding a sphinx plot gallery to the `astroplan` docs.
I haven't used this, but I think it's very easy to do:
http://sphinx-gallery.readthedocs.org/en/latest/auto_examples/index.html#examples-index

I wasn't sure if it's a good idea at all, and if so it's probably for later (although it could just as well be set up now), so I made a separate issue from #7 for discussion ...

**Comment by jberlanga** [comment]
@astroplanners/astroplanners-contributors 

I *really* like this idea. :+1:

**Comment by bmorris3** [comment]
:clap: :+1:

**Comment by adrn** [comment]
:+1:

**Comment by cdeil** [comment]
Sounds like we'll add a Sphinx gallery to `astroplan`.
Setting this up should be done in an independent PR from any PR implementing actual plots.
It should just contain one dummy plot for now and then we should see that this works locally, on travis-ci and readthedocs.

@jberlanga – Is this something you're interested in doing in the coming weeks or should I give it a try?

**Comment by eteq** [comment]
:+1: from me too, and I think it fits well with the other stuff @jberlanga will be working on (but it's her call).

**Comment by jberlanga** [comment]
@cdeil  Yeah, I can do this after I get the plotting code all straightened out (hopefully w/in the next week).

**Comment by cdeil** [comment]
@jberlanga – Thanks!
I've assigned this issue to you.
